File: backend/app/schemas/alert.py
import uuid
from datetime import datetime

from pydantic import BaseModel, ConfigDict


class AlertOut(BaseModel):
    model_config = ConfigDict(from_attributes=True)

    id: uuid.UUID
    company_id: uuid.UUID
    type: str
    entity_type: str
    entity_id: uuid.UUID
    message: str
    severity: str
    status: str
    created_at: datetime


class UnreadCountOut(BaseModel):
    count: int

File: frontend/src/types/alert.ts
export type AlertSeverity = 'baja' | 'media' | 'alta'
export type AlertStatus = 'no_leida' | 'leida'

export interface Alert {
  id: string
  company_id: string
  type: string
  entity_type: string
  entity_id: string
  message: string
  severity: AlertSeverity
  status: AlertStatus
  created_at: string
}
